### DISCUSSION TITLE
Cannot find 'ApplicationDelegateProxy' in scope while building iOS app

### DISCUSSION THREAD

**Discussion by antonioberetini** [ionic-team/capacitor#4310] [Q&A]
We tried generating the iOS application via capacitor, but we run into a build time error with code:

`Cannot find 'ApplicationDelegateProxy' in scope`

![image](https://user-images.githubusercontent.com/65619488/110472535-1842fd00-80a3-11eb-80eb-bfc084c90d6a.png)

We have followed the steps mentioned here in the docs:
https://capacitorjs.com/docs/v3/ios

Any idea we might be running into this issue?

**Comment by jcesarmobile** [comment]
You mean https://capacitorjs.com/docs/v3/updating/3-0?

Did you first run this?
```
npm install @capacitor/ios@next
npx cap sync ios
```
